lib/flipper/api/v1/actions/groups_gate.rb

```diff
@@ -0,0 +1,49 @@
+require 'flipper/api/action'
+require 'flipper/api/v1/decorators/feature'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class GroupsGate < Api::Action
+          route %r{api/v1/features/[^/]*/groups/?\Z}
+
+          def post
+            ensure_valid_params
+            feature = flipper[feature_name]
+            feature.enable_group(group_name)
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          def delete
+            ensure_valid_params
+            feature = flipper[feature_name]
+            feature.disable_group(group_name)
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          private
+
+          def ensure_valid_params
+            json_response({ code: 100, message: 'Feature not found.', more_info: '' }, 404) unless feature_names.include?(feature_name)
```

Comment:
you're totally right I had same thoughts about it looking like an http status code. I was thinking it would be nice to have a similar concept to http status code meaning where 100 level codes could be for problem with feature, 200 level error could be problem with gate, etc. but I like the idea of keeping it simple